apcd-cms/src/apps/exception/templates/exception_submission_form/exception_threshold_form.html

```diff
@@ -209,43 +207,98 @@ <h4>Acknowledgment of Terms</h4>
+        <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+        {# Adds blank space listener to all text input fields on page load #}
+        <script>
           const formTextInputs = document.querySelectorAll('input[type=text], textarea');
-          noEmptyInputs(formTextInputs);
-          </script> 
-          <!-- To require exception is not longer than a year -->
-            <script id="form-exception_length_less_than_year" type="module">
-              $(function(){
-                  var dtToday = new Date();
-                  
-                  var month = dtToday.getMonth() + 1;
-                  var day = dtToday.getDate();
-                  var year = dtToday.getFullYear();
-                  if(month < 10)
-                      month = '0' + month.toString();
-                  if(day < 10)
-                      day = '0' + day.toString();
-                  
-                  var minDate = dtToday.toISOString().substr(0, 10);
-                  $('#expiration-date').attr('min', minDate);
-                  year = year + 1;
-                  var maxDate = year + '-' + month + '-' + day;    
-                  $('#expiration-date').attr('max', maxDate);
+          noEmptyInputs(formTextInputs);          
+        </script> 
+        <!-- To require exception is not longer than a year -->
+        <script id="form-exception_length_less_than_year" type="module">
+          $(function () {
+            var dtToday = new Date();
+
+            var month = dtToday.getMonth() + 1;
+            var day = dtToday.getDate();
+            var year = dtToday.getFullYear();
+            if (month < 10)
+              month = '0' + month.toString();
+            if (day < 10)
+              day = '0' + day.toString();
+
+            var minDate = dtToday.toISOString().substr(0, 10);
+            $('#expiration-date').attr('min', minDate);
+            year = year + 1;
+            var maxDate = year + '-' + month + '-' + day;
+            $('#expiration-date').attr('max', maxDate);
+          });
+        </script>
+        <!-- To fetch CDL codes on file type input change -->
+        <script>
+          const fileTypeSelect = document.querySelector('#file_type');
+          const fieldTypeSelect = document.querySelector('#field-threshold-exception');
+          const helpText = document.getElementById('help-text-threshold-requested')
+          fileTypeSelect.addEventListener('change', function (event) {
+            /*Clears help text when file type changes*/
+            helpText.innerHTML = '';
+            selectedFileType = event.target.value;
+            fieldTypeSelect.innerHTML = '';
+
+          });
+        </script>
+        <script>
+          $(document).ready(function() {
+            $('#file_type').change(function() {
+              var file_type = $(this).val();
+              $.ajax({
+                url: '/submissions/threshold-exception',
```

Comment:
```suggestion
                url: '/submissions/threshold-exception/',
```
Resolves 301 redirect response this call first gets on firing it (the pattern without the trailing `/` is not defined in `urls.py`

Reply:
> Looks good, working well!! I had one small change requested, and one comment:
> 
> The Expiration Date field doesn't clear on re-selecting the file type from the dropdown - should it? I could see it not being important to clear it, don't know if it matters.

@edmondsgarrett Changes made! I don't think the date needs to clear, the helper text and field type clear is to prevent field types showing from the last, and therefore incorrect, file type selection. I can change this though if you think that would be better UI